cmd/fs-v1-multipart.go

```diff
@@ -744,7 +744,13 @@ func (fs fsObjects) CompleteMultipartUpload(bucket string, object string, upload
 		if removeObjectDir {
 			basePath := pathJoin(fs.fsPath, minioMetaMultipartBucket, bucket)
 			derr := fsDeleteFile(basePath, pathJoin(basePath, object))
-			errorIf(derr, "unable to remove %s in %s", pathJoin(basePath, object), basePath)
+			if derr = errorCause(derr); derr != nil {
+				// dont print the error errFileNotFound, as it is expected that concurrent
+				// complete-multipart-uploads would have deleted the file, and it wont be found
```

Comment:
This comment is not clear.  It can be like
```
//  In parallel execution of CompleteMultipartUpload could have deleted temporary state files/directory, it is safe to ignore errFileNotFound
```

Reply:
done